packs/data/the-slithering-bestiary.db/aspis-technician.json

```
@@ -1332,7 +1419,7 @@
                     "value": null
                 },
                 "description": {
-                    "value": "<p>The Aspis technician deals an extra 2d6 precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>The Aspis technician deals an extra [[/r 2d6[precision]]] precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```

```suggestion
                    "value": "<p>The Aspis technician deals an extra 2d6 precision precision damage to flat-footed creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
```
We don't add inlines for precision damage unless they have an associated damage type.